examples/pulse_control/4_Bell_pair_with_pulses_Rigetti.ipynb

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
<p>1) "a<s>n</s> Hadamard gate"</p><p> </p><p>2) Specifically<s> here</s>, we will focus ... of the Rigetti<s>'s</s> device"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>